www/src/css/snaplist.css

```diff
@@ -34,6 +44,10 @@
   margin-top:140%;
 }
 
+.b-store .b-snaplist_grid .b-snaplist__item:before {
```

Comment:
OK, I get the point that it can be easier to control the layout by changing one parent class between grid/list.

In such case I'm OK with leaving it nested, because making it better BEM in CSS will mean complicating JS code.
I would just suggest in cases like this adding a line of comment about why the nesting is needed.

Also, I'm not sure why we need 3 nested classes? Shouldn't something like `.b-snaplist_grid .b-snaplist__item` and `.b-snaplist_list .b-snaplist__item` be enough to have different styles for grid/list options?